go.mod

```diff
@@ -22,24 +22,34 @@ require (
 	github.com/topfreegames/extensions v8.1.0+incompatible
 	github.com/topfreegames/go-extensions-kafka v1.1.0
 	github.com/topfreegames/protos v1.6.1
-	google.golang.org/grpc v1.31.0
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.34.0
+	go.opentelemetry.io/otel v1.9.0
+	google.golang.org/grpc v1.48.0
+)
+
+require (
+	go.opentelemetry.io/otel/exporters/jaeger v1.9.0
```

Comment:
I believe we don't need this anymore, right?

Reply:
Done!